### PR TITLE
Readme Updated

### PR DESCRIPTION
**Pull request by sampoearas**
Usage Steps updated; Added line for 3rd optional import example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,15 @@ This project and the following release notes have been migrated from the old Ara
 
 Release | Notes
 --------|--------
+[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Readme usage updated
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Readme usage updated. Tested on 11 SP12, 11 SP15.
```

File: README.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ This project and the following release notes have been migrated from the old Ara
...
 [v2](https://github.com/ArasLabs/generic-action-item/releases/tag/v2) | Generic Action Item Solution package and Documentation for Aras 10 + 11
 [v1](https://github.com/ArasLabs/generic-action-item/releases/tag/v1) | Generic Action Item Solution package and Documentation
 
 #### Supported Aras Versions
 
 Project | Aras
 --------|------
+[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Tested on 11 SP12, 15
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | 10 SPx, 11 SPx
```

File: README.md [hunk 3]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
 10. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\2-Import - Example on DCO (optional)\2-Import\imports (admin).mf`.
-11. Close the Aras Package Import tool.
+11. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\3-Import - Example Action Item Template (optional)\imports (admin).mf`.
+12. Close the Aras Package Import tool.
 
 ## Usage
-
-See [generic Action Item Concepts v1-2(A10).pdf](./Documentation/generic%20Action%20Item%20Concepts%20v1-2(A10).pdf) for more information on using this project.
+This package can be used one of two ways:
+* by use of an Action Item Template set up by an Administrator
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
* Use an Action Item Template set up by an Administrator
```

File: README.md [hunk 4]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
+> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.
+
+###Example Usage: 
+1. (Optional) Create Template(s) for action items
+2. Create New or Open Existing Express DCO
+  * if new, must save item before using template action
+3. Use Action to add from template or add ad hoc using tab
+4. Fire workflow for action using RMB action in search grid/tab
+5. Complete action items via inbasket (closing item will require user password)
+  * if notes are needed to close, add to action item via RMB view command
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
  * If notes are needed to close, add to action item via RMB view command.
```

File: README.md [hunk 5]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
+This package can be used one of two ways:
+* by use of an Action Item Template set up by an Administrator
+* Added to an existing DCO item by a user either as ad-hoc or by using an existing template as above
+
+> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.
+
+###Example Usage: 
+1. (Optional) Create Template(s) for action items
+2. Create New or Open Existing Express DCO
+  * if new, must save item before using template action
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
  * If the item is new, save item before using template action.
```

File: README.md [hunk 6]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
-11. Close the Aras Package Import tool.
+11. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\3-Import - Example Action Item Template (optional)\imports (admin).mf`.
+12. Close the Aras Package Import tool.
 
 ## Usage
-
-See [generic Action Item Concepts v1-2(A10).pdf](./Documentation/generic%20Action%20Item%20Concepts%20v1-2(A10).pdf) for more information on using this project.
+This package can be used one of two ways:
+* by use of an Action Item Template set up by an Administrator
+* Added to an existing DCO item by a user either as ad-hoc or by using an existing template as above
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
* Add to an existing DCO item by a user, either as ad-hoc or by using an existing template as above.
```

File: README.md [hunk 7]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
+
+> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.
+
+###Example Usage: 
+1. (Optional) Create Template(s) for action items
+2. Create New or Open Existing Express DCO
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
2. Create or open an Express DCO.
```

File: README.md [hunk 8]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
-
-See [generic Action Item Concepts v1-2(A10).pdf](./Documentation/generic%20Action%20Item%20Concepts%20v1-2(A10).pdf) for more information on using this project.
+This package can be used one of two ways:
+* by use of an Action Item Template set up by an Administrator
+* Added to an existing DCO item by a user either as ad-hoc or by using an existing template as above
+
+> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.
+
+###Example Usage: 
+1. (Optional) Create Template(s) for action items
```
Comment (EliJDonahue) [inline README.md]:
```suggestion
1. (Optional) Create Template(s) for action items.
```

File: README.md [hunk 9]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
+2. Create New or Open Existing Express DCO
+  * if new, must save item before using template action
+3. Use Action to add from template or add ad hoc using tab
```
Comment (EliJDonahue) [inline README.md:66]: Try to be very specific about how people need to do things. Which action? How do they run that action? Is it a toolbar button, main menu action, or context menu action?

**Review by EliJDonahue — CHANGES_REQUESTED** [review]
It sounds like there may actually be two different use cases for the Usage section - template vs. ad hoc. I think it would make sense to list the steps for both use cases separately. See [the MBOM Export project](https://github.com/ArasLabs/mbom-export) for an example of a project with two use cases.

File: README.md (hunk 9 above)
Comment (sampoearas) [inline README.md:66]:
```suggestion
3. Use Actions > Add Action Item From Template to add from template or create a new relationship to an ad hoc item using the Action Item tab (Both methods can be used together)
```

File: README.md [hunk 10]
```diff
@@ -47,11 +49,26 @@ Project | Aras
...
+4. Fire workflow for action using RMB action in search grid/tab
```
Comment (sampoearas) [inline README.md:67]:
```suggestion
4. Start Workflow for action using RMB action in the Action Item admin search grid or the Action Item tab on a related item
```

**Review by sampoearas — COMMENTED** [review]
Added updates for further review.

File: README.md (hunk 9 above)
Comment (EliJDonahue) [reply README.md:66]: 👍

File: README.md (hunk 10 above)
Comment (EliJDonahue) [reply README.md:67]: 👍

**Review by EliJDonahue — APPROVED** [review]
👍

**Comment by EliJDonahue** [comment]
@sampoearas - Go ahead a commit the suggestions you made (I don't have write access to your fork) and then merge this PR.

**Pull request merged**